nsis/SectionCoreReqs.nsh

```diff
@@ -31,11 +31,11 @@
 	File                                            ..\bin\PCSX2_keys.ini.default
 
 	!insertmacro UNINSTALL.LOG_CLOSE_INSTALL
-
-    SetOutPath "$INSTDIR\Cheats"
+	
+	;/bin/Cheats needs CreateDirectory to prevent MakeNSIS from failing due to the folder being empty
     !insertmacro UNINSTALL.LOG_OPEN_INSTALL
-    File                                            ..\bin\Cheats\*
-    !insertmacro UNINSTALL.LOG_CLOSE_INSTALL
+    CreateDirectory                                 "$INSTDIR\Cheats"
```

Comment:
The cheats directory in the install folder isn't used anymore, so there's no need to create it.